lib/logidze/ignore_log_data.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Logidze
+  # Add `has_logidze` method to AR::Base
+  module IgnoreLogData
+    extend ActiveSupport::Concern
+
+    module BackportIgnoredColumns # :nodoc:
+      attr_writer :ignored_columns
+
+      def all(*args)
+        super(*args).select(attribute_names - ignored_columns)
+      end
+
+      def ignored_columns
+        @ignored_columns ||= []
+      end
+    end
+
+    def self.included(base)
+      base.extend BackportIgnoredColumns if Rails::VERSION::MAJOR < 5
+      base.ignored_columns += ["log_data"]
+    end
+
+    module ClassMethods # :nodoc:
+      def with_log_data
+        select(attribute_names + ["log_data"])
+      end
+    end
+
+    def log_data
```

Comment:
Do we need to override this? If the column is ignored (in Rails 5), would Rails raise this exception for us?

I’d like to use as many Rails built-in functionality as possible